### PR TITLE
Feature/Stylesheet refactor

### PR DESCRIPTION
**Pull request by loicsans**
Fixed remaining button issues + 
Fixed language selector displacement and removed from public page.

### CODE REVIEW

**Comment by loicsans** [comment]
Takes care of this: https://github.com/akvo/akvo-flow/issues/1920#issuecomment-276641034 (do not close the issue please)
and this: https://github.com/akvo/akvo-flow/issues/1869 (expect point 2. which needs to be addressed in a global structural improvement)

File: Dashboard/app/js/templates/navSurveys/nav-surveys-main.handlebars [hunk 1]
```diff
@@ -65,6 +65,8 @@
       {{/if}}
     </section>
 
+
```
Comment (muloem) [inline Dashboard/app/js/templates/navSurveys/nav-surveys-main.handlebars]: accidental whitespace?

File: Dashboard/app/css/main.scss [hunk 2]
```diff
@@ -4180,7 +4181,11 @@ div.dataCollectedDate label input {
     border-top: none;
 }
 
-.publicPage {}
+.publicPage {
```
Comment (loicsans) [inline Dashboard/app/css/main.scss]: That's fine

File: Dashboard/app/js/templates/navSurveys/nav-surveys-main.handlebars (hunk 1 above)
Comment (loicsans) [inline Dashboard/app/js/templates/navSurveys/nav-surveys-main.handlebars]: yes

**Pull request merged**